src/driver/kvm.rs

```diff
@@ -1,17 +1,54 @@
 use kvmi::{
     kvm_regs, kvm_segment, KVMIntrospectable, KVMiCr, KVMiEvent, KVMiEventReply, KVMiEventType,
-    KVMiInterceptType,
+    KVMiInterceptType, KVMiPageAccess,
 };
+use std::convert::TryFrom;
 use std::convert::TryInto;
 use std::error::Error;
 use std::mem;
 use std::vec::Vec;
 
 use crate::api::{
-    CrType, Event, EventReplyType, EventType, InterceptType, Introspectable, Registers, SegmentReg,
-    X86Registers, PAGE_SHIFT,
+    Access, CrType, Event, EventReplyType, EventType, InterceptType, Introspectable, Registers,
+    SegmentReg, X86Registers, PAGE_SHIFT,
 };
 
+impl TryFrom<Access> for u8 {
```

Comment:
```suggestion
impl Into<u8> for Access {
```

Reply:
Again, we should really use `KVMiPageAccess` instead of `u8` everywhere possible so the conversion will be unambiguous.